common/constants/shared.ts

```diff
@@ -56,6 +58,10 @@ export const observabilityIntegrationsID = 'integrations';
 export const observabilityIntegrationsTitle = 'Integrations';
 export const observabilityIntegrationsPluginOrder = 9020;
 
+export const observabilityDatasourcesID = 'datasources';
+export const observabilityDatasourcesTitle = 'Datasources';
+export const observabilityDatasourcesPluginOrder = 9030;
```

Comment:
In this PR we should to add `data sources` to the overview page: https://playground.opensearch.org/app/opensearch_management_overview

Reply:
Done in: [5cc9387](https://github.com/opensearch-project/dashboards-observability/pull/967/commits/5cc9387af97f79ca81c722555d926d0d2521f7c8)